app/models/solidus_afterpay/gateway.rb

```diff
@@ -12,5 +12,9 @@ def initialize(options)
         config.user_agent = SolidusAfterpay::UserAgentGenerator.new(merchant_id: options[:merchant_id]).generate
       end
     end
+
+    def authorize(_amount, _payment_source, _gateway_options)
+      ActiveMerchant::Billing::Response.new(true, 'Transaction approved')
```

Comment:
Correct me if I'm wrong but this is probably never called if you're using the immediate payment flow and configure Solidus to capture payments right away, right?